src/netex_idf/lines.rs

```diff
@@ -95,12 +99,22 @@ fn load_netex_lines(
 fn make_lines(lines_netex_idf: &CollectionWithId<LineNetexIDF>) -> Result<CollectionWithId<Line>> {
     let mut lines = CollectionWithId::default();
     for ln in lines_netex_idf.values() {
+        let commercial_mode_id = if let Some(commercial_mode_id) =
+            MODES.get::<str>(&ln.mode).map(|m| {
+                let (cid, _) = m.commercial_mode;
+                cid.to_string()
+            }) {
+            commercial_mode_id
+        } else {
+            warn!("{} not found", ln.mode);
```

Comment:
this would never happen.
You already test if mode exist https://github.com/CanalTP/transit_model/pull/386/files#diff-dc95a9a382c6c3104f734c584c7468f5R80